dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -102,6 +102,12 @@
     "NOTE: This size is a per worker (i.e., per GPU) configuration, "
     "and not cluster-wide!",
 )
+@click.option(
+    "--rmm-pool-async/--no-rmm-pool-async",
```

Comment:
I chose to go with a flag for this option - if we think there might be other resources down the line we also want to support, a string input may be a better option.

Reply:
Yeah, this is why I suggested the string idea. For now we can keep it like this and extend it in the future.